upgrade_test.py

```diff
@@ -23,6 +22,36 @@
 from sdcm.nemesis import RollbackNemesis
 from sdcm.nemesis import UpgradeNemesis
 from sdcm.fill_db_data import FillDatabaseData
+from functools import wraps
+
+
+def truncate_entries(func):
+    @wraps(func)
+    def _inner(self, *args, **kwargs):
+        # Perform validation of truncate entries in case the new version is 3.1 or more
+        node = args[0]
+        new_version = self.params.get('new_version', default='')
```

Comment:
You can't use this parameter for check which version we are going to upgrade to, in our CI we never use this parameter.
 we are using only the new_scylla_repo.
the `new_scylla_version` was added a way to select formal versions, but in CI we pass down non formal versions.

We need to think of a better way to asses to which version we are going to upgrade.

Reply:
> You can't use this parameter for check which version we are going to upgrade to, in our CI we never use this parameter.
> we are using only the new_scylla_repo.
> the `new_scylla_version` was added a way to select formal versions, but in CI we pass down non formal versions.
> 
> We need to think of a better way to asses to which version we are going to upgrade.

I didn't found other way to recognize that. Do you have idea?

Reply:
As agreed with @bentsi  and @fruch , added SCT_TAGRET_UPGRADE_VERSION parameter.
vars/rollingUpgradePipeline.groovy is updated accordingly